Vagrantfile

```diff
@@ -0,0 +1,19 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+rake_target = ENV['TEST'] ? "test TEST=#{ENV['TEST']}" : "test"
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.network "private_network", type: "dhcp"
+  config.vm.provision "shell", inline: <<-SHELL
+    sudo apt-get install software-properties-common
```

Comment:
Should we use  `apt-get install -y` in this line?
This line works for `ubuntu/trusty64`, though.